R/dynamic.R

```diff
@@ -74,17 +74,18 @@ jekyll = function(
   build_all = function() knit_maybe(input, output, script, method = 'jekyll')
 
   in_dir(dir, {
-    if (!serve) build_all()
-    jekyll_build()
```

Comment:
Another issue is when you have a brand-new site, the `_site` dir does not exist initially, so there is nothing to serve.